### PR TITLE
CRX What's New updates

### PR DESCRIPTION
**Pull request by dotproto**
Add two new items to the extensions What's New page

* The recent CWS analytics revamp blog post
* Identity API promise support

### CODE REVIEW

**Comment by dotproto** [comment]
Tagging in @rachelandrew rather than @jpmedley as I believe she indicated that she's handling extensions related changes until we have a dedicated tech writer. Please correct me if I'm wrong 😄

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -15,6 +15,25 @@ updated: 2022-06-15
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Chrome 105: Promises for the Identity API
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:18]:
```suggestion
### Chrome 105: promises for the Identity API
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 2]
```diff
@@ -15,6 +15,25 @@ updated: 2022-06-15
...
+July 15, 2022
+
+Functions on the [Identity API](/docs/extensions/reference/identity/#method-getAuthToken) now
+support promise based calls. This comes with a slight change to the surface for
+[`identity.getAuthToken()`](/docs/extensions/reference/identity/#method-getAuthToken), where the
+asynchronous return set to a promise based call will have "token" and "grantedScopes" as parameters
+on a single object (as opposed to the callback version receiving them as separate arguments to the
+callback).
+
+### Blog post: Chrome Web Store Analytics Revamp {: #cws-analytics-revamp }
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:29]:
```suggestion
### Blog post: Chrome Web Store analytics revamp {: #cws-analytics-revamp }
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 3]
```diff
@@ -15,6 +15,25 @@ updated: 2022-06-15
...
+
+July 28, 2022
+
+Chrome Web Store has a revamped item analytics experience for the Chrome Web Store Developer
+Dashboard. The new dashboard is easier to understand at a glance and consolidates the most useful
+information upfront. [Read the blog post](/blog/cws-analytics-revamp/) for more information.
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:35]:
```suggestion
information up front. [Read the blog post](/blog/cws-analytics-revamp/) for more information.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by Duca1976** [comment]
???

Il gio 4 ago 2022, 23:45 Joe Medley ***@***.***> ha scritto:

**Comment by dotproto** [comment]
@jpmedley, could you take another pass at this?

**Review by jpmedley — APPROVED** [review]

**Pull request merged**